index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>A Web Page</title>
+  <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+  <header class="largeView"><h1>Login</h1></header>
+  <section class="largeView">
+    <form>
+      <label for="email">Email</label>
+      <br>
+      <input type="email" name="email" required autofocus>
+      <br>
+      <label for="password">Password</label>
+      <br>
+      <input type="password" name="psw" minlength="7" required>
```

Comment:
The `for` attribute relates to the `name` of an input element -- so, you'd want this to be `password`.